frappe_er_generator/frappe_er_generator/er_generator.py

```diff
@@ -136,7 +144,10 @@ def get_table(data, link_list, doctypes, child_tables, amended_from):
             else:
                 table_element_list.append(
                     f'<tr><td port="{field.get("fieldname")}">{field.get("label")}</td></tr>')
-        if field.get("fieldtype") == "Link" and (amended_from or field.get("fieldname") != "amended_from"):
+        if field.get("fieldtype") == "Link" and (
+        omit_links is None or 
+        (omit_links == 'all' and field.get("options") != data.get("name")) or
```

Comment:
Again this code have problem.
for eg: I want to omit fieldname `email` from doctype `User` but that field `email` exist in multiple doctype , so according to this code `email` field will be omited in each table where it exist.

Reply:
That is the intended functionality. I just want to be able to remove amended_from fields while keeping everything else. So then it should be able to specify, per doctype, what fields to omit? Including an 'all' option for both doctype and docfield?